db/schema_tables.hh

```diff
@@ -112,6 +112,7 @@ static constexpr auto VIEWS = "views";
 static constexpr auto TYPES = "types";
 static constexpr auto FUNCTIONS = "functions";
 static constexpr auto AGGREGATES = "aggregates";
+static constexpr auto SCYLLA_AGGREGATES = "scylla_aggregates";
```

Comment:
The whole thing needs to be protected with a cluster feature / schema feature so a partially upgraded cluster doesn't break.

Reply:
?

Reply:
This is commit that add SCYLLA_AGGREGATES schema feature

https://github.com/scylladb/scylla/pull/10295/commits/cb4b22a869f1db3b61e64d06c5e748a47ac6f04d